chandra_aca/tests/test_maude_decom.py

```diff
@@ -272,6 +274,56 @@ def test_partial_images():
         assert np.all(table[i]["IMG"].mask == mask[table[i]["IMGTYPE"]])
 
 
+def test_aca_images_chunks_1():
+    """Test that a fetch longer than the maude step limit works"""
+    single_fetch_limit = maude_decom.MAUDE_SINGLE_FETCH_LIMIT
+    maude_decom.MAUDE_SINGLE_FETCH_LIMIT = 1 * u.min
+    start = "2023:001:00:00:01.000"
+    stop = "2023:001:00:05:01.000"
+    tstart = CxoTime(start).secs
+    tstop = CxoTime(stop).secs
+    try:
+        imgs = maude_decom.get_aca_images(start, stop)
+    finally:
+        maude_decom.MAUDE_SINGLE_FETCH_LIMIT = single_fetch_limit
+
+    imgs.sort(["TIME", "IMGNUM"])
+
+    for slot in range(8):
+        # Confirm that the data is basically contiguous
+        ok_slot = imgs["IMGNUM"] == slot
+        # Confirm no duplicates by VCDUCTR
+        assert len(np.unique(imgs[ok_slot]["VCDUCTR"])) == len(imgs[ok_slot])
+        # Confirm for these 8x8 data that there's no gap > 5 seconds
+        assert np.max(np.diff(imgs[ok_slot]["TIME"])) < 5
+
+    # Confirm that the returned data times are within the start stop
+    assert imgs["TIME"][0] >= tstart
+    assert imgs["TIME"][-1] <= tstop
+
+    # Confirm that the beginning and end match the expected values
+    assert abs(imgs[0]["TIME"] - tstart) < 2
+    assert abs(imgs[-1]["TIME"] - tstop) < 2
+
+    imgs_start = maude_decom.get_aca_images(start, CxoTime(start) + 60 * u.s)
+    imgs_stop = maude_decom.get_aca_images(CxoTime(stop) - 60 * u.s, stop)
+    assert np.all(imgs[0] == imgs_start[0])
+    assert np.all(imgs[-1] == imgs_stop[-1])
```

Comment:
I simplified the PR and removed the code I had to fetch extra time and re-filter the images by TIME.  And now this isn't passing.

Reply:
I fixed this.

Reply:
Doesn't this suggest an upstream sort fix in get_aca_images?

Reply:
The upstream fix would be to sort by time and slot. That's a question of API / expectations. In the current release the output is empirically sorted by [IMGNUM, TIME], so all the slot=0 are first and slot=7 are last. For chunked data this pattern is repeated for each chunk.

Overall I think we don't care as long as the images are sorted by TIME for a given IMGNUM. That is the only ordering that is currently guaranteed.

Reply:
Right - though we added sorting on the mica side to be out.sort(keys=["TIME", "IMGNUM"]), so maybe maude_decom.get_aca_images should just do the same for consistency.